lib/benchee/conversion/scale.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Benchee.Conversion.Scale do
+  @moduledoc """
+  Functions for scaling values to other units. Different domains handle
+  this task differently, for example durations and counts.
+
+  See `Benchee.Conversion.Count` and `Benchee.Conversion.Duration` for examples
+  """
+
+  @type unit :: atom
```

Comment:
Couldn't we reuse the unit type definiton from `Format` here or well the other way around (Format uses the type definition from Scale) ?